fundingmanager.go

```diff
@@ -266,6 +285,14 @@ type fundingConfig struct {
 	// discovered short channel ID of a formerly pending channel to outside
 	// sub-systems.
 	ReportShortChanID func(wire.OutPoint, lnwire.ShortChannelID) error
+
+	// ZombieSweeperInterval is the periodic time interval in which the zombie
+	// sweeper is run
+	ZombieSweeperInterval time.Duration
+
+	// ReservationTimeout is the length of idle time that must pass before a
+	// reservation is considered a zombie
```

Comment:
nit: use complete sentences for comments!

Reply:
Fixed.